### ISSUE TITLE
"cannot parse fractional part in..." when importing csv data where a number has a dot but no decimal digits

### ISSUE THREAD

**Issue by fluxion-t** [VictoriaMetrics/VictoriaMetrics#3544]
**Description of the bug**

When importing CSV data, if a metric has a decimal dot but without decimal digits, like 20., the line is not imported.

**To Reproduce**

This process is done under Debian Bullseye.

```
str="test,2022-12-25T16:57:12+01:00,10,20.,30"
curl -v -d "$str" 'http://localhost:8428/api/v1/import/csv?format=1:label:mytest,2:time:rfc3339,3:metric:M10,4:metric:M20,5:metric:M30'
```

The output is the following:

```
*   Trying ::1:8428...
* connect to ::1 port 8428 failed: Connection refused
*   Trying 127.0.0.1:8428...
* Connected to localhost (127.0.0.1) port 8428 (#0)
> POST /api/v1/import/csv?format=1:label:mytest,2:time:rfc3339,3:metric:M10,4:metric:M20,5:metric:M30 HTTP/1.1
> Host: localhost:8428
> User-Agent: curl/7.74.0
> Accept: */*
> Content-Length: 40
> Content-Type: application/x-www-form-urlencoded
> 
* upload completely sent off: 40 out of 40 bytes
* Mark bundle as not supporting multiuse
< HTTP/1.1 204 No Content
< X-Server-Hostname: bcn-verda
< Date: Mon, 26 Dec 2022 17:53:35 GMT
< 
* Connection #0 to host localhost left intact
```

but at /var/syslog we find the following:

```
Dec 26 18:53:35 bcn-verda victoria-metrics-prod[653]: 2022-12-26T17:53:35.752Z#011error#011VictoriaMetrics/lib/protoparser/csvimport/parser.go:125#011error when parsing csv line "test,2022-12-25T16:57:12+01:00,10,20.,30": cannot parse metric value for "M20" from "20.": cannot parse fractional part in "20."; skipping this line
```

**Expected behavior**

I expected the API to accept a number like 20. with the same exact meaning as 20.0

**Version**
```
$ ./victoria-metrics-prod --version
victoria-metrics-20221214-210650-tags-v1.85.1-0-g09a70d3e9
```

**Used command-line flags**
./victoria-metrics-prod -storageDataPath=/path/to/victoria-metrics

**Comment by valyala** [comment]
@fluxion-t , thanks for the bugreport! The bug has been fixed in the commit c90752a8be2ecf3624ee80de30b5ebb30da3b6c7 . This commit will be included in the next release of VictoriaMetrics. In the meantime you can build VictoriaMetrics from this commit according to [these build instructions](https://docs.victoriametrics.com/#how-to-build-from-sources)

**Comment by valyala** [comment]
VictoriaMetrics should properly accept values without fractional part and/or without integer parts such as `123.` or `.234` during data import starting from [v1.86.0](https://github.com/VictoriaMetrics/VictoriaMetrics/releases). Closing the issue as fixed.

**Comment by valyala** [comment]
FYI, the bugfix for this issue has been included also into [VictoriaMetrics v1.79.7 LTS release](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.79.7).
